integr8ly/installation
# Hide identity providers rather than deleting them

## Additional Information
Instead of deleting the htpasswd identity provider we are just going to hide it from the login screen instead.

This way the users in that provider can be used for the install/uninstall without issue.

Related update to CI/CD: https://github.com/integr8ly/ci-cd/pull/47

## Verification
- Start a *fresh* cluster from RHPDS (it's unlikely this will work on a cluster that has old installations run on it - without some manual shaking of the cluster)
- Run the install
- Check `/etc/origin/master/master-config.yaml` has `login: false` for the htpasswd provider, and `login: true` for the sso provider.
- login on `oc` cli with opentlc-mgr account.
- at web login confirm no htpasswd login option
- login to ui with admin@example.com account
- run uninstall with `-e openshift_login=true -e openshift_username=<opentlc-mgr> -e openshift_password=<opentlc-mgr password> -e openshift_master_public_url=<your cluster url>`
*N.B.* This is not normally required, but is required to validate these changes.
- Check `/etc/origin/master/master-config.yaml` that the htpasswd has `login: true`
- Login on CLI and web with opentlc-mgr account.
